test/hard.test.js

```diff
@@ -17,6 +18,7 @@ describe('hard', () => {
     stylelint
       .lint({
         files: [fixture],
+        syntax: 'scss',
```

Comment:
We removed this in #170 as it isn't actually a stylelint feature apparently

Reply:
No, it is only not in `.stylelintrc` as #170 did, but it exists in [Node.js API](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/node-api.md#syntax).

Reply:
Ahh okay